### ISSUE TITLE
MESOP_CONCURRENT_UPDATES_ENABLED  - concurrent streaming requests has odd behavior maybe

### ISSUE THREAD

**Issue by richard-to** [google/mesop#981]
Need to look into this one more to determine if this is a big issue or not. Could be minor. I'm also not sure the reason for the issue yet.

I was testing out the animation demo. Basically I clicked "Transform Color" followed by "Fade In / Fade Out". I also tested out the reverse.

What happens in this scenario is that the second animation that is clicked will not fully finish. The final rendering will be some intermediate state.

**Comment by wwwillchen** [comment]
Good catch - it looks like what's happening (when I inspect the network tab) is that when there's two concurrent event streams, whichever finishes first, will also close the second one, which makes sense because: https://github.com/google/mesop/blob/129e0d011ebb315d98457ab723c185ee1dab823e/mesop/web/src/services/channel.ts#L118

I'm not sure what's the right way of solving this, we could have a map of eventSource (instead of having a single one, it's actually kind of interesting how the concurrent updates even works with the single eventSource instance) or we can have some kind of counter to keep track of the pending requests and still use a single eventSoruce.

**Comment by richard-to** [comment]
This env var is deprecated, so closing.
